src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -116,10 +116,17 @@ function ExportAnnotations({
     }),
     [exportableAnnotations],
   );
-  const [selectedUser, setSelectedUser] = useState(
-    // Try to preselect current user
-    userList.find(userInfo => userInfo.userid === currentUser) ??
+
+  // Try to preselect current user
+  const [selectedUserId, setSelectedUserId] = useState(currentUser);
+  // Re-compute selectedUser if:
+  //  - `userList` changes: This means annotations where created/deleted/updated
+  //  - `selectedUserId` changes: This means user was manually changed
+  const selectedUser = useMemo(
```

Comment:
This variable name is a bit confusing because it sounds like the selected user ID, but it is actually metadata and annotations for them. The simplest change would be to rename this `selectedUserAnnotations` to clearly contrast with `selectedUserId`.